vars/buildPlugin.groovy

```diff
@@ -41,7 +41,7 @@ def call(Map params = [:]) {
         boolean addToolEnv = !useContainerAgent
 
         if(useContainerAgent && (label == 'linux' || label == 'windows')) {
-            String agentContainerLabel = jdk == '8' ? 'maven' : 'maven-11'
+            def agentContainerLabel = jdk == '8' ? 'maven' : 'maven-' + jdk
```

Comment:
or

```groovy
            def agentContainerLabel = jdk == '8' ? 'maven' : "maven-$jdk"
```

Reply:
🤔 what are the pro and cons of these 2 concatenation methods (in this context, e.g. pipeline library)?

Reply:
None, purely stylistic.

Reply:
In pure Groovy there is no difference, but in Pipeline the latter is subject to `InterpolatedSecretsAction` while the former is not (see #210).

Reply:
Oh I forgot about https://github.com/jenkins-infra/pipeline-library/pull/210#issuecomment-854852071